src/main/resources/swagger/api-docs.yaml

```diff
@@ -6726,6 +6726,61 @@ paths:
                   $ref: '#/components/schemas/ToolClass'
       x-passthrough: true
       x-passthrough-target: agora
+  /servicePerimeters/{servicePerimeterName}/projects/{projectName}:
+    put:
+      tags:
+        - Service Perimeters
+      summary: Add a project to a service perimeter
+      description: |
+        Add a project to a service perimeter, must have 'add_to_service_perimeter'
+        action on project and 'add_project' action on perimeter, included in owner
+        role for both
+      operationId: addProjectToServicePerimeter
+      parameters:
+        - name: servicePerimeterName
+          in: path
+          description: |
+            Fully qualified google service perimeter name in the form of
+            accessPolicies/[POLICY NUMBER]/servicePerimeters/[NAME], url encoded
```

Comment:
Who is encoding this parameter? I believe we could avoid requiring an encoded perimeter name by doing something like this in the route:

```
path("servicePerimeters" / Segment.repeat(4, Slash) / "projects" / Segment) { (servicePerimeterName, projectName) =>

    val qualifiedName = servicePerimeterName.mkString("/")
    ...
````

I realize that this may have upstream and downstream implications so if we _have_ to work with a url-encoded parameter then consider this totally optional

Reply:
Interesting!  I had not seen that feature.

I'd rather keep it as-is since we'd have to re-encode to send it on to Rawls anyway.  Unless we are OK with making a breaking change to the Rawls API too.  Doesn't seem worth it to me in this case.